examples/text_labels_and_annotations/dfrac_demo.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Feb 25 17:50:37 2017
```

Comment:
Remove this date; we have `git log` for that. Can you please modify the docstring to follow the [sphinx-gallery format](http://matplotlib.org/devdocs/devel/MEP/MEP12.html#clean-up-guidelines)?